android/src/main/java/com/amplitude/android/Configuration.kt

```diff
@@ -39,9 +40,14 @@ open class Configuration @JvmOverloads constructor(
     var locationListening: Boolean = true,
     var flushEventsOnClose: Boolean = true,
     var minTimeBetweenSessionsMillis: Long = MIN_TIME_BETWEEN_SESSIONS_MILLIS,
-    @Deprecated("Please use 'defaultTracking.sessions' instead.", ReplaceWith("defaultTracking.sessions"))
+    @Deprecated("Please use 'autocapture' instead and set 'AutocaptureOptions.SESSIONS' to enable the option.")
+    /** The SDK will no longer track changes to `trackingSessionEvents`'s value after initialization. */
     var trackingSessionEvents: Boolean = true,
+    @Suppress("DEPRECATION")
+    @Deprecated("Please use 'autocapture' instead", ReplaceWith("autocapture"))
+    /** The SDK will no longer track changes to the `defaultTracking` options after initialization. */
     var defaultTracking: DefaultTrackingOptions = DefaultTrackingOptions(),
+    autocapture: Set<AutocaptureOption> = setOf(AutocaptureOption.SESSIONS),
```

Comment:
should we also support boolean, making autocapture = true enable all autocapture options?

Reply:
Based on the discussion we had on autocapture sync, having an ALL configuration for autocapture would make it hard for us to add new options later without surprising the customers.

Reply:
Good to know thanks.